news/563.bugfix

```diff
@@ -0,0 +1 @@
+Selecting a missing node from a ListConfig with `throw_on_missing` set to True now raises the intended exception.
```

Comment:
Function name here to be more specific about what you mean by "selecting"
```suggestion
OmegaConf.select() a missing node from a ListConfig with `throw_on_missing` set to True now raises the intended exception.
```

Reply:
Done in b9a5db6